gradle/build-logic/convention/src/main/java/app/tivi/gradle/utils/Android.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.tivi.gradle.utils
+
+import com.android.build.gradle.BaseExtension
+import org.gradle.api.JavaVersion
+import org.gradle.api.Project
+import org.gradle.api.artifacts.VersionCatalogsExtension
+import org.gradle.kotlin.dsl.configure
+import org.gradle.kotlin.dsl.dependencies
+import org.gradle.kotlin.dsl.getByType
+
+fun Project.configureAndroid() {
+    val libs = extensions.getByType<VersionCatalogsExtension>().named("libs")
+
+    android {
+
+        compileSdkVersion(libs.findVersion("compileSdk").get().requiredVersion.toInt())
+
```

Comment:
I don't really like this part, both the part where I assume `requiredVersion` is the one that should be used and the `toInt()` conversion. I think a better solution would be to move the SDK values out of the version catalog and put them in a class next to these convention plugins

```kotlin
object Versions {
    const val compileSdk = 33
    const val minSdk = 23
    const val targetSdk = 31
```

Reply:
Yeah, I like that idea! Much better when we have a centralised place